main.tf

```diff
@@ -98,6 +98,9 @@ module "servers" {
       "ha.yml" : templatefile("${path.module}/manifests/ha.yml.tpl", {
         operator_replica = local.operator_replica
       }),
+      "kube-vip.yml" : templatefile("${path.module}/manifests/kube-vip.yml.tpl", {
+        vip_address = "192.168.44.5"
```

Comment:
Can you try to switch the internal ip by this one? and see whether cilium works well?